packages/core/src/components/Drawer/Drawer.tsx

```diff
@@ -81,41 +88,61 @@ export const HvDrawer = (props: HvDrawerProps) => {
     onClose,
     anchor = "right",
     buttonTitle = "Close",
+    disableBackdropClick = false,
     ...others
   } = useDefaultProps("HvDrawer", props);
 
-  const { classes, cx } = useClasses(classesProp);
+  const { classes, cx, css } = useClasses(classesProp);
+  const { colors } = useTheme();
 
   const closeButtonDisplay = () => <Close role="none" />;
 
   const CloseButtonTooltipWrapper = buttonTitle
     ? withTooltip(closeButtonDisplay, buttonTitle, "top")
     : closeButtonDisplay;
 
+  const handleBackdropClick = useCallback(
+    (event) => {
+      if (!disableBackdropClick) {
+        onClose?.(event, "backdropClick");
+      }
+    },
+    [disableBackdropClick, onClose]
+  );
+
   return (
-    <MuiDrawer
-      className={cx(classes.root, className)}
-      id={id}
-      anchor={anchor}
-      open={open}
-      PaperProps={{
-        classes: {
-          root: classes.paper,
-        },
-      }}
-      onClose={onClose}
-      {...others}
-    >
-      <HvButton
-        id={setId(id, "close")}
-        className={classes.closeButton}
-        variant="secondaryGhost"
-        onClick={onClose}
-        aria-label={buttonTitle}
+    <>
+      <MuiDrawer
+        className={cx(classes.root, className)}
+        id={id}
+        anchor={anchor}
+        open={open}
+        PaperProps={{
+          classes: {
+            root: classes.paper,
+          },
+        }}
+        onClose={onClose}
+        {...others}
       >
-        <CloseButtonTooltipWrapper />
-      </HvButton>
-      {children}
-    </MuiDrawer>
+        <HvButton
+          id={setId(id, "close")}
+          className={classes.closeButton}
+          variant="secondaryGhost"
+          onClick={onClose}
+          aria-label={buttonTitle}
+        >
+          <CloseButtonTooltipWrapper />
+        </HvButton>
+        {children}
+      </MuiDrawer>
+      <MuiBackdrop
+        open={!!open}
+        onClick={handleBackdropClick}
+        className={css({
+          background: hexToRgbA(colors?.atmo4 || theme.colors.atmo4, 0.8),
```

Comment:
Same as above - the fallback doesn't do anything. Also, `0.8` is already the default
```suggestion
          backgroundColor: hexToRgbA(colors?.atmo4),
```

We should also add `classes.background` here - it's currently unused 🤔